RELEASE.md

```diff
@@ -0,0 +1,105 @@
+Info: Updating values file to default to carvel support.
+Info: Generating the json-schema for the chart and converting to yaml
+Info: Copying README to version directory.
+Info: Generating image lock file for Kubeapps 8.0.14
+Info: Collecting all images to /home/michael/dev/vmware/package-for-kubeapps/build/images.txt
+Info: Generating fake deployments for kbld images.
+...
+Info: Generating /home/michael/dev/vmware/package-for-kubeapps/8.0.14-dev1/package.yaml
+creating: /home/michael/dev/vmware/package-for-kubeapps/8.0.14-dev1/package.yaml
+Info: Pushing projects-stg.registry.vmware.com/kubeapps/kubeapps:8.0.14-dev1 image.
+Info: Testing installation of new package 8.0.14-dev1
+Creating cluster "kubeapps-carvel-e2e" ...
+ ✓ Ensuring node image (kindest/node:1.21.1) 🖼
+ ✓ Preparing nodes 📦
+ ✓ Writing configuration 📜
+ ✓ Starting control-plane 🕹️
+ ...
+Package install reconciled successfully. Deleting...
+| Uninstalling package 'kubeapps' from namespace 'default'
+...
+Deleting cluster "kubeapps-carvel-e2e" ...
+Info: Skipping creation of release for staging test.
+Info: Finished. To test the package manually (until automated tests) you can make the package available on your cluster with:
+Info: kubectl apply -n kapp-controller-packaging-global -f ./metadata.yaml -f ./8.0.14-dev1/package.yaml
+Info: and install with:
+Info: tanzu package install kubeapps --package-name kubeapps.community.tanzu.vmware.com --version 8.0.14-dev1
+```
+
+Given that the above was a non-production run against the staging registry, the generated local files will not be committed or tagged nor will a GitHub release have been created. You can delete the generated files to put your local git repository back to its initial state:
+
+```bash
+rm 8.0.14-dev1 -rf
+```
+
+For a production release you will need to run the script with the `-p` option which, in addition to pushing the image to the production registry, will also add, commit and tag the new files locally, push the tag to the `upstream` remote and create the release. The script does not currently push your updated branch with the new commits upstream, see [Issue 20](https://github.com/vmware-tanzu/package-for-kubeapps/issues/20).
+
+## Updating the TCE repository to include the new package
+
+Once we've published a new package to the production registry, we need to request that the new package be included in the TCE repository via a pull-request.
+
+The [initial pull-request to include Kubeapps in the TCE repository](https://github.com/vmware-tanzu/community-edition/pull/4666) added:
+
+- `addons/packages/kubeapps/metadata.yaml` - the metadata which tends not to change with new versions,
+- `addons/packages/kubeapps/vendir.yml` - a vendir configuration which specifies the GitHub release from which to pull the README.md and `package.yaml` and where to place them.
+
+The vendir tool was then run in the `addons/packages/kubeapps` directory to ensure all files are in place, before committing and creating the PR.
```

Comment:
Is this something to be added to a new script maybe?

Reply:
Possibly yep, I'd wait until the we know the cost is worth the effort (we've done it once so far).